Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Kimai - Time tracking/Main application](https://hosted.weblate.org/projects/kimai/messages/).


It also includes following components:

* [Kimai - Time tracking/User feedback (Flash messages and Alerts)](https://hosted.weblate.org/projects/kimai/flashmessages/)

* [Kimai - Time tracking/Plugins](https://hosted.weblate.org/projects/kimai/plugins/)

* [Kimai - Time tracking/Export](https://hosted.weblate.org/projects/kimai/export/)

* [Kimai - Time tracking/Validations](https://hosted.weblate.org/projects/kimai/validations/)

* [Kimai - Time tracking/Tasks](https://hosted.weblate.org/projects/kimai/tasks/)

* [Kimai - Time tracking/Reporting](https://hosted.weblate.org/projects/kimai/reporting/)

* [Kimai - Time tracking/Invoice-Calculator](https://hosted.weblate.org/projects/kimai/invoice-calculator/)

* [Kimai - Time tracking/Invoice-Templates](https://hosted.weblate.org/projects/kimai/invoice-templates/)

* [Kimai - Time tracking/Date-Picker](https://hosted.weblate.org/projects/kimai/daterangepicker/)

* [Kimai - Time tracking/Teams](https://hosted.weblate.org/projects/kimai/teams/)

* [Kimai - Time tracking/Kiosk](https://hosted.weblate.org/projects/kimai/kiosk/)

* [Kimai - Time tracking/System configurations](https://hosted.weblate.org/projects/kimai/system-configuration/)

* [Kimai - Time tracking/Wizard](https://hosted.weblate.org/projects/kimai/wizard/)

* [Kimai - Time tracking/Emails](https://hosted.weblate.org/projects/kimai/emails/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/kimai/messages/horizontal-auto.svg)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/kimai/kimai?pullRequest=5254) <br/>All committers have signed the CLA.

weblate committed:
Translated using Weblate (Tamil)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (German)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (German)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (78 of 78 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (17 of 17 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (German)

Currently translated at 100.0% (39 of 39 strings)

Translated using Weblate (German)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (485 of 485 strings)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Translated using Weblate (Tamil)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Tamil)

Currently translated at 97.4% (38 of 39 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (8 of 8 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (27 of 27 strings)

Translated using Weblate (Tamil)

Currently translated at 98.7% (77 of 78 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Tamil)

Currently translated at 95.4% (463 of 485 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (13 of 13 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (4 of 4 strings)

Translated using Weblate (Tamil)

Currently translated at 82.3% (14 of 17 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (10 of 10 strings)

Translated using Weblate (Tamil)

Currently translated at 100.0% (2 of 2 strings)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Update translation files

Updated by "Remove blank strings" hook in Weblate.

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Added translation using Weblate (Tamil)

Translated using Weblate (Croatian)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Spanish)

Currently translated at 99.5% (483 of 485 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (French)

Currently translated at 94.1% (16 of 17 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (485 of 485 strings)

Translated using Weblate (Hebrew)

Currently translated at 99.1% (481 of 485 strings)

Translated using Weblate (Ukrainian)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Croatian)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Finnish)

Currently translated at 98.7% (479 of 485 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Polish)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Italian)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (Hebrew)

Currently translated at 98.9% (480 of 485 strings)

Translated using Weblate (French)

Currently translated at 98.7% (479 of 485 strings)

Translated using Weblate (Spanish)

Currently translated at 99.3% (482 of 485 strings)

Translated using Weblate (German)

Currently translated at 99.5% (483 of 485 strings)

Translated using Weblate (English)

Currently translated at 100.0% (485 of 485 strings)

Co-authored-by: Biscuittttt <biscuitwithtea.tall310@passinbox.com>
Co-authored-by: Carlos Carreras <mytriponlinux@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Kevin Papst <kevin@kevinpapst.de>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Milo Ivir <mail@milotype.de>
Co-authored-by: Wellington Terumi Uemura <wellingtonuemura@gmail.com>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: abdelbasset jabrane <ribago9317@cubene.com>
Co-authored-by: yblis <yblisss@yahoo.fr>
Co-authored-by: Максим Горпиніч <mgorpinic2005@gmail.com>
Co-authored-by: தமிழ்நேரம் <anishprabu.t@gmail.com>
Translate-URL: https://hosted.weblate.org/projects/kimai/daterangepicker/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/emails/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/export/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/flashmessages/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-calculator/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/invoice-templates/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/kiosk/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ar/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/en/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/es/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fi/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/fr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/he/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/hr/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/it/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pl/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/messages/uk/
Translate-URL: https://hosted.weblate.org/projects/kimai/plugins/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/reporting/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/system-configuration/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/tasks/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/teams/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/de/
Translate-URL: https://hosted.weblate.org/projects/kimai/validations/ta/
Translate-URL: https://hosted.weblate.org/projects/kimai/wizard/ta/
Translation: Kimai - Time tracking/Date-Picker
Translation: Kimai - Time tracking/Emails
Translation: Kimai - Time tracking/Export
Translation: Kimai - Time tracking/Invoice-Calculator
Translation: Kimai - Time tracking/Invoice-Templates
Translation: Kimai - Time tracking/Kiosk
Translation: Kimai - Time tracking/Main application
Translation: Kimai - Time tracking/Plugins
Translation: Kimai - Time tracking/Reporting
Translation: Kimai - Time tracking/System configurations
Translation: Kimai - Time tracking/Tasks
Translation: Kimai - Time tracking/Teams
Translation: Kimai - Time tracking/User feedback (Flash messages and Alerts)
Translation: Kimai - Time tracking/Validations
Translation: Kimai - Time tracking/Wizard
